py/janitor/site/templates/layout.html

```diff
@@ -23,18 +26,18 @@
         <script src="/_static/chart.js"></script>
         <script src="/_static/janitor.js"></script>
         {% include "analytics.html" ignore missing without context %}
```

Comment:
This can probably be dropped - it's inherited from the original (debian-janitor-specific) site.